
[2023, 9, 16, 12, 29, 22] :;: 26152 :;: ['  File "<stdin>", line 1, in <module>\n', '  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load\n', '  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked\n', '  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked\n', '  File "<frozen importlib._bootstrap_external>", line 940, in exec_module\n', '  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\vm.py", line 11, in <module>\n    import modules.audio as audio\n', '  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load\n', '  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked\n', '  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked\n', '  File "<frozen importlib._bootstrap_external>", line 940, in exec_module\n', '  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 1321, in <module>\n    log.crash(os.getpid())\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 27, in crash\n    log.data.append([pytools.clock.getDateTime(), str(strf), str(traceback.format_stack())])\n']
[2023, 9, 16, 15, 14, 7] :;: Uncaught exception :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 90, in exception_handler\n    log.crash("Uncaught exception")\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 27, in crash\n    log.data.append([pytools.clock.getDateTime(), str(strf), str(traceback.format_stack())])\n']
[2023, 9, 16, 15, 14, 7] :;: Type: :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 91, in exception_handler\n    log.crash(\'Type:\', exc_type)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 27, in crash\n    log.data.append([pytools.clock.getDateTime(), str(strf), str(traceback.format_stack())])\n']
[2023, 9, 16, 15, 14, 7] :;: <class 'AttributeError'> :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 91, in exception_handler\n    log.crash(\'Type:\', exc_type)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 27, in crash\n    log.data.append([pytools.clock.getDateTime(), str(strf), str(traceback.format_stack())])\n']
[2023, 9, 16, 15, 14, 7] :;: Value: :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 92, in exception_handler\n    log.crash(\'Value:\', exc_value)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 7] :;: 'builtin_function_or_method' object has no attribute 'instance' :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 92, in exception_handler\n    log.crash(\'Value:\', exc_value)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 7] :;: Traceback: :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 93, in exception_handler\n    log.crash(\'Traceback:\', exc_traceback)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 7] :;: <traceback object at 0x00000256F4D7E980> :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 93, in exception_handler\n    log.crash(\'Traceback:\', exc_traceback)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 13] :;: Uncaught exception :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 90, in exception_handler\n    log.crash("Uncaught exception")\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 13] :;: Type: :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 91, in exception_handler\n    log.crash(\'Type:\', exc_type)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 13] :;: <class 'voicemeeter.errors.VMRDriverError'> :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 91, in exception_handler\n    log.crash(\'Type:\', exc_type)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 13] :;: Value: :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 92, in exception_handler\n    log.crash(\'Value:\', exc_value)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 13] :;: VMR#VBVMR_GetParameterFloat() returned -1 :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 92, in exception_handler\n    log.crash(\'Value:\', exc_value)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 13] :;: Traceback: :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 93, in exception_handler\n    log.crash(\'Traceback:\', exc_traceback)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 16, 15, 14, 13] :;: <traceback object at 0x00000256F4D7EBC0> :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 93, in exception_handler\n    log.crash(\'Traceback:\', exc_traceback)\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']
[2023, 9, 30, 10, 49, 40] :;: Audio Engine Instance Has Finished. :;: ['  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 79, in exit_handler\n    log.crash("Audio Engine Instance Has Finished.")\n', '  File "C:\\Users\\joshp\\Desktop\\deffered_audio_client\\modules\\audio.py", line 43, in crash\n    pytools.IO.appendFile(".\\\\logs\\\\errors\\\\" + log.dateString + "\\\\event_" + log.timeString + ".log", "\\n" + str(dateArray) + " :;: " + str(strf) + " :;: " + str(traceback.format_stack()).replace("\\n", "    \\\\n\\t"))\n']